test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua

```diff
@@ -0,0 +1,46 @@
+--  Requirement summary:
+--  [RegisterAppInterface] REJECT
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that App will be rejected with HMI type WEB_VIEW 
+--  when mobile application has no policy record in local policy table.
+--
+--  Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to SDL.
+--
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table is empty.
```

Comment:
@ydementieiev Really strange expectation as for me. Why SDL must clear LPT on registration of application?
I think it is make sense to check that SDL do not add any permissions into LPT for application which tried to register here. 
But it requires to extend test scenario with new test step (triggfer PTU and check policy permissions for application which tried to register in PTS)

Reply:
@aderiabin check please [changes ](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/33588521b5f1ff45678b21bd871324afc6f42bd0)